method-info/winsorisation/winsorisation.md

```diff
@@ -0,0 +1,423 @@
+
+#### Expansion Estimation Calculations
+
+Used when values for auxiliary and calibration factor are **not** supplied.
+
+A _weight_ **w** is calculated for each row of data, for expansion estimation this is the supplied design weight
+
+$$ w = \text{design weight} $$
+
+When _weight_ **w** == 1 the marker is set as "NW_FE" and the row is removed from further calculations for Winsorisation.
+
+For a _weight_ **w** not equal to 1, where _target values_ are **y** the _mean target values_, **$\bar{y}$** are caluculated as:
+
+$$ \bar{y} = \frac{i}{n} \sum_{i=1}^{n} y_i $$
+
+Using this, a _threshold_ **k** can be calculated to determine whether a _target value_ is an outlier or not:
+
+$$ k = \bar{y} + (\frac{L}{w - 1}) $$
+
+#### Ratio Estimation Calculations
+
+Used when values for auxiliary and calibration factor **are** supplied.
+
+A _weight_ **w** is calculated for each row of data
+
+$$ w = \text{calibration factor} * \text{design weight} $$
+
+When _weight_ **w** <= 1 the marker is set as "NW_AG" and the row is removed from further calculations for Winsorisation.
+
+For a _weight_ **w** > 1 the method calculates:
+
+_weighted target_ **ay**
+
+$$ay = \text{design weight} * \text{target value}$$
+
+_sum of weighted targets_ $sy$ is calculated:
+
+$$sy = \sum ay $$
+
+_weighted auxilary_ **ax**
+
+$$ ax = \text{design weight} * auxiliary $$
+
+_sum of weighted auxiliary_ $sx$ is calculated:
+
+$$ sx = \sum ax $$
+
+The _weighted ratio of target and auxiliary_ **wta** is calculated as:
+
+$$ wta = \frac{sy}{sx} $$
+
+A _unit_predict_value_ **mu**, is calculated as:
+
+$$ mu = auxiliary * wta $$
+
+Once these calculations are made a _threshold_ **k** can be calculated to determine whether a _target value_ is an outlier or not:
+
+$$ k = mu + (\frac{L}{w - 1}) $$
+
+### Outlier Weight Calculation
+
+The threshold, **k** derived from either Expansion or Ratio Estimation is then used to determine whether a _target value_ exceeds the computed threshold. Values that exceed the threshold have an _outlier weight_, **o** calculated that is the ratio a value can be reduced by to bring it closer to the other target values in the group being analysed. It is this _outlier weight_ value that the Winsorisation method returns.
+
+For each row of data the _target value_ is compared with the calculated _threshold_, **k** and a _modified target value_ **$ y^* $** is computed:
+
+When _target value_, **$y$** is <= _threshold_, **k** the _modified target_ equals the _target value_ :
+
+$$ y^* = y $$
+
+When _target value_, **$y$** is > _threshold_, **k** the _modified target_ is calculated as :
+
+$$ y^* = \frac{y}{w} +(1-(\frac{1}{w}))*k $$
+
+The _outlier weight_, **$o$** can then be calculated as:
```

Comment:
should it be **o** or should the example above be like this one?

Reply:
I am not sure what your comment means, the calculation of **o** is essentially the modified target value divided by the original value, if there isn't a modification to the target value then it becomes the original value divided by the original value which equates to 1. The algebra is ok as far as I can see.